src/test.rs

```diff
@@ -328,7 +328,13 @@ fn clone_reference_tree(
 #[test]
 #[ignore]
 fn test_dry_runs() {
-    let (setup_cs, finalization_hint) = init_or_synth_cs_for_sha256::<DevCSConfig, true>(None);
+    let (setup_cs, finalization_hint) =
+        init_or_synth_cs_for_sha256::<DevCSConfig, Global, true>(None);
+    let (proving_cs, _) =
+        init_or_synth_cs_for_sha256::<ProvingCSConfig, Global, true>(finalization_hint.as_ref());
+    let witness = proving_cs.witness.unwrap();
+    let (reusable_cs, _) =
+        init_or_synth_cs_for_sha256::<ProvingCSConfig, Global, false>(finalization_hint.as_ref());
```

Comment:
does this just correct something Dima missed (adding the Global allocator generic parameter)?
If so, why did you move them?

Reply:
Changes were needed because of dima´s changes to boojum´s API. The move was not intended but should not make a difference.